system76_acpi.c

```diff
@@ -292,6 +454,9 @@ static void system76_notify(struct acpi_device *acpi_dev, u32 event)
 	case 0x84:
 		kb_led_hotkey_color(data);
 		break;
+	case 0x85:
+		input_key(data, KEY_SCREENLOCK);
```

Comment:
Is `KEY_SCREENLOCK` right? I would have assumed `KEY_DISPLAYTOGGLE`.

Reply:
KEY_SCREENLOCK is used to lock the screen on the addw2, effectively implementing the screen off key (Fn-F2).
Fn-F7 is already implemented by sending Super+P, which is what the proprietary EC does.